### PR TITLE
Use dependabot to update docker image version

### PR DESCRIPTION
**Pull request by DanielFran**
Related with:
- https://github.com/jhipster/generator-jhipster/pull/20012
- https://github.com/jhipster/generator-jhipster/pull/20014
---

Please make sure the below checklist is followed for Pull Requests.

- [ ] [All continuous integration tests](https://github.com/jhipster/generator-jhipster/actions) are green
- [ ] Tests are added where necessary
- [ ] The JDL part is updated if necessary
- [ ] [jhipster-online](https://github.com/jhipster/jhipster-online) is updated if necessary
- [ ] Documentation is added/updated where necessary
- [ ] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/generator-jhipster/blob/main/CONTRIBUTING.md) are followed

When you are still working on the PR, consider converting it to Draft (below reviewers) and adding `skip-ci` label, you can still see CI build result at your branch.

### CODE REVIEW

**Comment by DanielFran** [comment]
@mshima was not able to update snapshots

```> npm run update-snapshots
npm WARN config global `--global`, `--local` are deprecated. Use `--location=global` instead.

> generator-jhipster@7.9.3 update-snapshots
> UPDATE_SNAPSHOT=all mocha test generators --no-insight --forbid-only

'UPDATE_SNAPSHOT' is not recognized as an internal or external command,
operable program or batch file.

**Comment by mshima** [comment]
@DanielFran looks like you are running on windows.
The current script uses unix environment variable.
- We can pass using arguments but it requires sequential test instead of parallel.
- Use the archived cross-env.
- Create a new mocha plugin for update snapshots.

File: generators/docker-compose/templates/docker-compose.yml.ejs [hunk 1]
```diff
@@ -39,7 +39,7 @@ services:
 <%_ } _%>
 <%_ if (serviceDiscoveryEureka) { _%>
   jhipster-registry:
-    image: <%= DOCKER_JHIPSTER_REGISTRY %>
+    image: <%- dockerContainers.jhipster-registry %>
```
Comment (mshima) [inline generators/docker-compose/templates/docker-compose.yml.ejs:42]:
```suggestion
    image: <%- dockerContainers['jhipster-registry'] %>
```
or
```suggestion
    image: <%- dockerContainers.jhipsterRegistry %>
```
We can convert to camelcase

File: generators/docker-compose/templates/docker-compose.yml.ejs [hunk 2]
```diff
@@ -121,7 +121,7 @@ services:
     ports:
       - 9090:9090
   alertmanager:
-    image: <%= DOCKER_PROMETHEUS_ALERTMANAGER %>
+    image: <%- dockerContainers.prometheus-alertmanager %>
```
Comment (DanielFran) [inline generators/docker-compose/templates/docker-compose.yml.ejs:124]:
```suggestion
    image: <%- dockerContainers.prometheusAlertmanager %>
```

File: generators/kubernetes/templates/registry/consul-config-loader.yml.ejs [hunk 3]
```diff
@@ -33,7 +33,7 @@ spec:
     spec:
       containers:
       - name: consul-config-loader
-        image: <%= DOCKER_CONSUL_CONFIG_LOADER %>
+        image: <%- dockerContainers.consul-config-loader %>
```
Comment (DanielFran) [inline generators/kubernetes/templates/registry/consul-config-loader.yml.ejs:36]:
```suggestion
        image: <%- dockerContainers.consulConfigLoader %>
```

File: generators/kubernetes/templates/registry/jhipster-registry.yml.ejs [hunk 4]
```diff
@@ -82,7 +82,7 @@ spec:
       terminationGracePeriodSeconds: 10
       containers:
       - name: jhipster-registry
-        image: <%= DOCKER_JHIPSTER_REGISTRY %>
+        image: <%- dockerContainers.jhipster-registry %>
```
Comment (DanielFran) [inline generators/kubernetes/templates/registry/jhipster-registry.yml.ejs:85]:
```suggestion
        image: <%- dockerContainers.jhipsterRegistry %>
```

File: generators/openshift/templates/monitoring/jhipster-metrics.yml.ejs [hunk 5]
```diff
@@ -334,7 +334,7 @@ objects:
                 path: config.yml
           containers:
           - name: ${AM_APP_NAME}
-            image: <%= DOCKER_PROMETHEUS_ALERTMANAGER %>
+            image: <%- dockerContainers.prometheus-alertmanager %>
```
Comment (DanielFran) [inline generators/openshift/templates/monitoring/jhipster-metrics.yml.ejs:337]:
```suggestion
            image: <%- dockerContainers.prometheusAlertmanager %>
```

File: generators/openshift/templates/registry/consul.yml.ejs [hunk 6]
```diff
@@ -190,7 +190,7 @@ objects:
         spec:
           containers:
           - name: ${APPLICATION_NAME}-consul-config-loader
-            image: <%= DOCKER_CONSUL_CONFIG_LOADER %>
+            image: <%- dockerContainers.consul-config-loader %>
```
Comment (DanielFran) [inline generators/openshift/templates/registry/consul.yml.ejs:193]:
```suggestion
            image: <%- dockerContainers.consulConfigLoader %>
```

File: generators/openshift/templates/registry/jhipster-registry.yml.ejs [hunk 7]
```diff
@@ -80,7 +80,7 @@ objects:
           terminationGracePeriodSeconds: 10
           containers:
           - name: ${APPLICATION_NAME}
-            image: <%= DOCKER_JHIPSTER_REGISTRY %>
+            image: <%- dockerContainers.jhipster-registry %>
```
Comment (DanielFran) [inline generators/openshift/templates/registry/jhipster-registry.yml.ejs:83]:
```suggestion
            image: <%- dockerContainers.jhipsterRegistry %>
```

File: generators/server/templates/gradle/docker.gradle.ejs [hunk 8]
```diff
@@ -18,7 +18,7 @@
 -%>
 jib {
     from {
-        image = "<%= DOCKER_JAVA_JRE %>"
+        image = "<%- dockerContainers.java-jre %>"
```
Comment (DanielFran) [inline generators/server/templates/gradle/docker.gradle.ejs:21]:
```suggestion
        image = "<%- dockerContainers.javaJre %>"
```

File: generators/server/templates/pom.xml.ejs [hunk 9]
```diff
@@ -159,7 +159,7 @@
         <modernizer-maven-plugin.version>2.4.0</modernizer-maven-plugin.version>
         <jacoco-maven-plugin.version><%= JACOCO_VERSION %></jacoco-maven-plugin.version>
         <jib-maven-plugin.version><%- javaDependencies['jib-maven-plugin'] %></jib-maven-plugin.version>
-        <jib-maven-plugin.image><%= DOCKER_JAVA_JRE %></jib-maven-plugin.image>
+        <jib-maven-plugin.image><%- dockerContainers.java-jre %></jib-maven-plugin.image>
```
Comment (DanielFran) [inline generators/server/templates/pom.xml.ejs:162]:
```suggestion
        <jib-maven-plugin.image><%- dockerContainers.javaJre %></jib-maven-plugin.image>
```

File: generators/server/templates/src/main/docker/consul.yml.ejs [hunk 10]
```diff
@@ -30,7 +30,7 @@ services:
     command: consul agent -dev -ui -client 0.0.0.0 -log-level=INFO
 
   consul-config-loader:
-    image: <%= DOCKER_CONSUL_CONFIG_LOADER %>
+    image: <%- dockerContainers.consul-config-loader %>
```
Comment (DanielFran) [inline generators/server/templates/src/main/docker/consul.yml.ejs:33]:
```suggestion
    image: <%- dockerContainers.consulConfigLoader %>
```

File: generators/server/templates/src/main/docker/jhipster-control-center.yml.ejs [hunk 11]
```diff
@@ -52,7 +52,7 @@
 name: <%= baseName.toLowerCase() %>
 services:
   jhipster-control-center:
-    image: '<%= DOCKER_JHIPSTER_CONTROL_CENTER %>'
+    image: '<%- dockerContainers.jhipster-control-center %>'
```
Comment (DanielFran) [inline generators/server/templates/src/main/docker/jhipster-control-center.yml.ejs:55]:
```suggestion
    image: '<%- dockerContainers.jhipsterControlCenter %>'
```

File: generators/server/templates/src/main/docker/jhipster-registry.yml.ejs [hunk 12]
```diff
@@ -20,7 +20,7 @@
 name: <%= baseName.toLowerCase() %>
 services:
   jhipster-registry:
-    image: <%= DOCKER_JHIPSTER_REGISTRY %>
+    image: <%- dockerContainers.jhipster-registry %>
```
Comment (DanielFran) [inline generators/server/templates/src/main/docker/jhipster-registry.yml.ejs:23]:
```suggestion
    image: <%- dockerContainers.jhipsterRegistry %>
```

File: generators/server/templates/src/main/docker/swagger-editor.yml.ejs [hunk 13]
```diff
@@ -20,6 +20,6 @@
 name: <%= baseName.toLowerCase() %>
 services:
   swagger-editor:
-    image: <%= DOCKER_SWAGGER_EDITOR %>
+    image: <%- dockerContainers.swagger-editor %>
```
Comment (DanielFran) [inline generators/server/templates/src/main/docker/swagger-editor.yml.ejs:23]:
```suggestion
    image: <%- dockerContainers.swaggerEditor %>
```

File: test/templates/compose/01-gateway/src/main/docker/app.yml [hunk 14]
```diff
@@ -19,7 +19,7 @@ services:
             - 3306:3306
         command: mysqld --lower_case_table_names=1 --skip-ssl --character_set_server=utf8mb4 --explicit_defaults_for_timestamp
     jhipster-registry:
-        image: <%= DOCKER_JHIPSTER_REGISTRY %>
+        image: <%- dockerContainers.jhipster-registry %>
```
Comment (DanielFran) [inline test/templates/compose/01-gateway/src/main/docker/app.yml:22]:
```suggestion
        image: <%- dockerContainers.jhipsterRegistry %>
```

File: test/templates/compose/02-mysql/src/main/docker/app.yml [hunk 15]
```diff
@@ -17,7 +17,7 @@ services:
             - 3306:3306
         command: mysqld --lower_case_table_names=1 --skip-ssl --character_set_server=utf8mb4 --explicit_defaults_for_timestamp
     jhipster-registry:
-        image: <%= DOCKER_JHIPSTER_REGISTRY %>
+        image: <%- dockerContainers.jhipster-registry %>
```
Comment (DanielFran) [inline test/templates/compose/02-mysql/src/main/docker/app.yml:20]:
```suggestion
        image: <%- dockerContainers.jhipsterRegistry %>
```

File: test/templates/compose/03-psql/src/main/docker/app.yml [hunk 16]
```diff
@@ -24,7 +24,7 @@ services:
             - 9200:9200
             - 9300:9300
     jhipster-registry:
-        image: <%= DOCKER_JHIPSTER_REGISTRY %>
+        image: <%- dockerContainers.jhipster-registry %>
```
Comment (DanielFran) [inline test/templates/compose/03-psql/src/main/docker/app.yml:27]:
```suggestion
        image: <%- dockerContainers.jhipsterRegistry %>
```

File: test/templates/compose/04-mongo/src/main/docker/app.yml [hunk 17]
```diff
@@ -8,13 +8,13 @@ services:
...
     mongodb:
-        image: <%= DOCKER_MONGODB %>
+        image: <%- dockerContainers.mongodb %>
         ports:
             - "27017:27017"
         # volumes:
         #     - ~/volumes/jhipster/msmongodb/mongodb/:/data/db/
     jhipster-registry:
-        image: <%= DOCKER_JHIPSTER_REGISTRY %>
+        image: <%- dockerContainers.jhipster-registry %>
```
Comment (DanielFran) [inline test/templates/compose/04-mongo/src/main/docker/app.yml:17]:
```suggestion
        image: <%- dockerContainers.jhipsterRegistry %>
```

File: test/templates/compose/05-cassandra/src/main/docker/app.yml [hunk 18]
```diff
@@ -37,7 +37,7 @@ services:
         volumes:
             - ../resources/config/cql:/cql:ro
     jhipster-registry:
-        image: <%= DOCKER_JHIPSTER_REGISTRY %>
+        image: <%- dockerContainers.jhipster-registry %>
```
Comment (DanielFran) [inline test/templates/compose/05-cassandra/src/main/docker/app.yml:40]:
```suggestion
        image: <%- dockerContainers.jhipsterRegistry %>
```

File: test/templates/compose/07-mariadb/src/main/docker/app.yml [hunk 19]
```diff
@@ -17,7 +17,7 @@ services:
             - 3306:3306
         command: mysqld --lower_case_table_names=1 --skip-ssl --character_set_server=utf8mb4 --explicit_defaults_for_timestamp
     jhipster-registry:
-        image: <%= DOCKER_JHIPSTER_REGISTRY %>
+        image: <%- dockerContainers.jhipster-registry %>
```
Comment (DanielFran) [inline test/templates/compose/07-mariadb/src/main/docker/app.yml:20]:
```suggestion
        image: <%- dockerContainers.jhipsterRegistry %>
```

File: test/templates/compose/10-couchbase/src/main/docker/app.yml [hunk 20]
```diff
@@ -22,7 +22,7 @@ services:
 #        volumes:
 #            - ~/volumes/jhipster/mscouchbase/couchbase/:/opt/couchbase/var
     jhipster-registry:
-        image: <%= DOCKER_JHIPSTER_REGISTRY %>
+        image: <%- dockerContainers.jhipster-registry %>
```
Comment (DanielFran) [inline test/templates/compose/10-couchbase/src/main/docker/app.yml:25]:
```suggestion
        image: <%- dockerContainers.jhipsterRegistry %>
```

File: test/templates/compose/11-mssql/src/main/docker/app.yml [hunk 21]
```diff
@@ -25,7 +25,7 @@ services:
             - 1433:1433
         command: /bin/bash -c '/opt/mssql/bin/sqlservr & echo "wait $$MSSQL_SLEEP sec for DB to start "; sleep $$MSSQL_SLEEP; /opt/mssql-tools/bin/sqlcmd -U sa -P $$SA_PASSWORD -d tempdb -q "EXIT(CREATE DATABASE $$MSSQL_DATABASE)"; wait;'
     jhipster-registry:
-        image: <%= DOCKER_JHIPSTER_REGISTRY %>
+        image: <%- dockerContainers.jhipster-registry %>
```
Comment (DanielFran) [inline test/templates/compose/11-mssql/src/main/docker/app.yml:28]:
```suggestion
        image: <%- dockerContainers.jhipsterRegistry %>
```

**Comment by DanielFran** [comment]
@mshima Can you update snapshots? I am still not able to do it on windows...

**Comment by mshima** [comment]
@DanielFran done.
Cherrypicked a commit from https://github.com/jhipster/generator-jhipster/pull/20212.

**Comment by DanielFran** [comment]
Thanks @mshima

**Pull request merged**